File: Backend/DTOs/FlightDto.cs
namespace Backend.DTOs
{
    public class FlightDto
    {
        public int FlightNumber { get; set; }
        public int DepartingAirportId { get; set; }
        public int DestinationAirportId { get; set; }
        public int AircraftId { get; set; }
        public DateTime DepartingTime { get; set; }
        public DateTime FlightTime { get; set; }
        public int FlightCost { get; set; }
        public int? DiscountId { get; set; }
    }

}

File: Frontend/src/app/app-logic/DTOs/flight-dto.ts
export interface FlightDto {
    flightNumber: number;
    departingAirportId: number;
    destinationAirportId: number;
    aircraftId: number;
    departingTime: Date;
    flightTime: Date;
    flightCost: number;
    discountId?: number;
}

